sunbeam-python/sunbeam/utils.py

```diff
@@ -72,18 +72,27 @@ def get_nic_macs(nic: str) -> list:
     return sorted([a["addr"] for a in addrs[netifaces.AF_LINK]])
 
 
+def filter_link_local(addresses: list[dict]) -> list[dict]:
```

Comment:
```suggestion
def filter_link_local(addresses: list[dict]) -> list[dict]:
```

This is python3.9 type hinting, py38 checks might fail

Reply:
Oh right, it's the snap and we're using py310 there, nvm